src/System.Collections.Concurrent/src/System/Collections/Concurrent/PartitionerStatic.cs

```diff
@@ -246,8 +246,9 @@ public static OrderablePartitioner<TSource> Create<TSource>(IEnumerable<TSource>
             int coreOversubscriptionRate = 3;
 
             if (toExclusive <= fromInclusive) throw new ArgumentOutOfRangeException(nameof(toExclusive));
-            int rangeSize = (toExclusive - fromInclusive) /
-                (PlatformHelper.ProcessorCount * coreOversubscriptionRate);
+            long substraction = (long)toExclusive - fromInclusive;
```

Comment:
Nit: "substraction" (which is mispelled anyway :) => "range"

Reply:
renamed, thank you for notice.